### DISCUSSION TITLE
經常出錯的字

### DISCUSSION THREAD

**Discussion by MooMeowCat** [tongwentang/tongwen-dict#19] [General]
姓余
剩餘，有餘
https://i.imgur.com/cQh7fMg.png

房舍
捨棄，捨卻
https://i.imgur.com/9nhxVTE.png
https://i.imgur.com/6qVwkmE.png

須得
鬍鬚，餘
https://i.imgur.com/NxzYmx1.png

**Comment by t7yang** [comment]
https://github.com/tongwentang/tongwen-dict/issues/20

**Comment by MooMeowCat** [comment]
念=心裏念著，念念不忘，顧念舊情，等等
唸=用口唸出，唸唸有詞，唸經，等等
https://i.imgur.com/lXTiBJG.png

**Comment by MooMeowCat** [comment]
幹乾
https://i.imgur.com/ATljtG1.png
